brew/src/main/java/nl/tudelft/serg/evosql/brew/data/Path.java

```diff
@@ -18,4 +18,7 @@
 
     /** The path number. */
     private final int pathNumber;
+
+    /** If path is a success from production query (returns something). */
+    private final boolean success;
```

Comment:
As a first skim in the code, I think you not only store this _success_ variable, but also what it returns. After all, in the generated tests, we have two options:

The first: the generated data, when processed by the production query, returns nothing. In this case, `assertTrue(result.isEmpty())`.

The second: the generated data, when processed by the production query, returns N rows. In this case, we should `assertEquals(N, result.size()` and, for each row, assert its content, e.g., `assertEquals(row1..n, result.row(1..n)`.

Does that make sense?

Reply:
Very good point, we're currently working on an implementation to handle this. 

For now we'd like to merge these changes first so that when we implement this our PR will not be too complex for review.